.github/workflows/stale.yaml

```diff
@@ -11,9 +11,10 @@ jobs:
         with:
           stale-issue-message: 'This issue has been automatically marked as stale because it has not had any activity in the last 60 days. Thank you for your contributions.'
           close-issue-message: 'This issue was closed because it has not had any activity in the last 120 days. Please reopen if you feel this is still valid.'
+          close-pr-message: "This pull request is being closed because it has not had any activity in the last 180 days. This is not a signal from maintainers that the PR has no value, it's just that innactive PRs are hard to come to completion! We appreciate the author's time and effort and encourage to re-open the pull request if there is intention to finish it!"
```

Comment:
```suggestion
          close-pr-message: "This pull request is being closed because it has not had any activity in the last 180 days. This is not a signal from the maintainers that the PR has no value. We appreciate the time and effort that you put into this work. If you're willing to re-open it, the maintainers will do their best to review it."
```

Reply:
PTAL :)